### PR TITLE
#10 Backend Implementation (Sign Up)

### PR DESCRIPTION
**Pull request by Electryk**
Waiting for #12 for testing

### CODE REVIEW

**Comment by Electryk** [comment]
Copy that

**Comment by nebur395** [comment]
I'm agree with @Gascon95

**Comment by Gascon95** [comment]
@Electryk your last commit is failing (the servlets can't find the DAOs and VOs, find why). Plus, the web.xml file is wrong. The path to the servlets should start on "servlet" directory and the endpoint should be "/signUp" instead of "/SignUp", as specified on the API documentation, which you can find on GoogleDrive.

**Comment by Gascon95** [comment]
@Electryk now Travis is passing, but remember to make the changes I pointed out earlier on the web.xml file.

**Comment by nebur395** [comment]
LGTM 👍

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -6,16 +6,16 @@
   version="3.0"
   metadata-complete="true">
 
-    <display-name>TrainingTracker</display-name>
+    <display-name>trainingTracker</display-name>
     <description>
-      TrainingTracker
+      trainingTracker
```
Comment (nebur395) [inline src/main/webapp/WEB-INF/web.xml]: I think this is not necessary.

**Pull request merged**